fabric-common-tags-api-v1/src/main/java/net/fabricmc/fabric/api/tag/v1/CommonEntityTypeTags.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.tag.v1;
+
+import net.minecraft.entity.EntityType;
+import net.minecraft.tag.TagKey;
+
+import net.fabricmc.fabric.impl.tag.common.TagRegistration;
+
+/**
+ * See {@link net.minecraft.tag.EntityTypeTags} for vanilla tags.
+ * Note that addition to some vanilla tags implies having certain functionality.
+ */
+public class CommonEntityTypeTags {
+	/**
+	 * Tag containing entity types that display a boss health bar.
+	 */
+	public static final TagKey<EntityType<?>> BOSSES = register("bosses");
+	public static final TagKey<EntityType<?>> MINECARTS = register("minecarts");
```

Comment:
imo, this one should be dynamically generated at runtime to automatically include any entity that extends `MinecartEntity`.